modules/openshift-developer-cli-odo-known-issues.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-release-notes.adoc
+
+[id="odo-known-issues_{context}"]
+= Known Issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760573[Bug 1760573] An active pointer is not pointing to a current active project.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760574[Bug 1760574] A deleted namespace is being listed in `odo project get` command.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760575[Bug 1760575] `odo app delete' removes application components but not services.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760577[Bug 1760577] `odo push` does not delete OpenShift objects when component name is changed.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760578[Bug 1760578] `odo watch` does not fail with error message for git based components but it should.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760580[Bug 1760580] `catalog list components` does not list out component of a specified namespace.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760583[Bug 1760583] `odo config unset` does not unset an environment variable but says it did.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760585[Bug 1760585] `odo delete --all` deletes the`$HOME/.odo` folder is deleted when run from `$HOME`.
```

Comment:
Missing space between "the" and "`$HOME/.odo`", so the backticks are rendering in the text.

And something is wrong with this sentence. Should it be...

```
deletes the`$HOME/.odo` folder when run from `$HOME`.
```